src/components/Row/Row.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person
+  onSelectPerson: (slug: string) => void
+  isSelected: boolean
+  motherSlug?: string
+  fatherSlug?: string
+};
+
+export const Row: React.FC<Props> = ({
+  person,
+  onSelectPerson,
+  isSelected,
+  motherSlug,
+  fatherSlug,
+}) => {
+  const {
+    slug,
+    motherName,
+    sex,
+    born,
+    died,
+    name,
+    fatherName,
+  } = person;
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({ 'has-background-warning': isSelected })}
+    >
+
+      <td>
+        <Link
+          onClick={() => onSelectPerson(slug)}
+          to={`#/people/:${person.slug}`}
+          className={motherSlug ? 'has-text-danger' : undefined}
```

Comment:
Use [classnames](https://www.npmjs.com/package/classnames) library to add classes conditionally